### DISCUSSION TITLE
Website: catch webgl disabled

### DISCUSSION THREAD

**Discussion by jborme** [nesbox/TIC-80#1689] [General]
I had a firefox profile with webgl.disabled=true and the cartridges wouldn't load. It is a small improvement for a corner case, but it would be great if the website would catch when the features it needs are disabled and displays the relevant message, the same as many websites display "please activate javascript". Just a suggestion. Thanks for the games and the fun.

**Comment by nesbox** [comment]
Created an issue #1692
Thanks
